sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/subscriptions/PubSubTestModels.java

```diff
@@ -39,6 +42,44 @@ public Action.Effect<Message> messageTwo(Message message) {
     }
   }
 
+  public static class SubscribeToTopicAction extends Action {
+
+    @Subscribe.Topic(value = "topicXYZ", consumerGroup = "cg")
+    public Action.Effect<Message> messageOne(Message message) {
+      return effects().reply(message);
+    }
+  }
+
+  public static class SubscribeToTwoTopicsAction extends Action {
+
+    @Subscribe.Topic("topicXYZ")
+    public Action.Effect<Message> methodOne(Message message) {
+      return effects().reply(message);
+    }
+
+    @Subscribe.Topic("topicXYZ")
+    public Action.Effect<Message2> methodTwo(Message2 message) {
+      return effects().reply(message);
+    }
+  }
+
+  public static class PublishToTopicAction extends Action {
+
+    @Publish.Topic("topicAlpha")
+    public Action.Effect<Message> messageOne(Message message) {
+      return effects().reply(message);
+    }
+  }
+
+  public static class RestWithPublishToTopicAction extends Action {
+
+    @PostMapping("/message/{msg}")
+    @Publish.Topic("foobar")
+    public Effect<Message> messageOne(@PathVariable String msg) {
+      return effects().reply(new Message(msg));
+    }
+  }
```

Comment:
Could it make sense to add an additional RestAnnotated-only method to one of these examples just to have a test asserting that such method gets ignored when it comes to the pubsub logic maybe?